<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UAV Mission Planner</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icons/32x32/gui/images/32x32/apps/spire.png</normaloff>:/icons/32x32/gui/images/32x32/apps/spire.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea|Qt::RightToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPlan_Flight"/>
   <addaction name="actionReset_Flight"/>
   <addaction name="actionTest_Flight"/>
   <addaction name="separator"/>
   <addaction name="actionPlace_Start_Point"/>
   <addaction name="actionPlace_Task_Area"/>
   <addaction name="separator"/>
   <addaction name="actionUAV_Parameters"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::RightToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave_Planning_Problem"/>
   <addaction name="separator"/>
   <addaction name="actionImport_Solution"/>
   <addaction name="actionExport_Solution"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionSave_Planning_Problem">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/48/gui/images/48/48px-Document-save.svg.png</normaloff>:/icons/48/gui/images/48/48px-Document-save.svg.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save Problem</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExport_Solution">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/48/gui/images/48/48px-Document-save.svg.png</normaloff>:/icons/48/gui/images/48/48px-Document-save.svg.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Export Solution</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+E</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/48/gui/images/48/48px-System-log-out.svg.png</normaloff>:/icons/48/gui/images/48/48px-System-log-out.svg.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/48/gui/images/48/48px-Document-open.svg.png</normaloff>:/icons/48/gui/images/48/48px-Document-open.svg.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open Problem</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionUAV_Parameters">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/48/gui/images/48/48px-Applications-system.svg.png</normaloff>:/icons/48/gui/images/48/48px-Applications-system.svg.png</iconset>
   </property>
   <property name="text">
    <string>UAV Parameters</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/48/gui/images/48/48px-Document-new.svg.png</normaloff>:/icons/48/gui/images/48/48px-Document-new.svg.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New Problem</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/16x16/gui/images/16x16/actions/fileclose.png</normaloff>:/icons/16x16/gui/images/16x16/actions/fileclose.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Close problem</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionImport_Solution">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/48/gui/images/48/48px-Document-open.svg.png</normaloff>:/icons/48/gui/images/48/48px-Document-open.svg.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Import Solution</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+I</string>
   </property>
  </action>
  <action name="actionPlan_Flight">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/32x32/gui/images/32x32/start or accept.png</normaloff>:/icons/32x32/gui/images/32x32/start or accept.png</iconset>
   </property>
   <property name="text">
    <string>Plan Flight</string>
   </property>
  </action>
  <action name="actionReset_Flight">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/32x32/gui/images/32x32/reset or undo.png</normaloff>:/icons/32x32/gui/images/32x32/reset or undo.png</iconset>
   </property>
   <property name="text">
    <string>Reset Flight</string>
   </property>
  </action>
  <action name="actionPlace_Start_Point">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/32x32/gui/images/32x32/startCircle.png</normaloff>:/icons/32x32/gui/images/32x32/startCircle.png</iconset>
   </property>
   <property name="text">
    <string>Place Start Point</string>
   </property>
  </action>
  <action name="actionPlace_Task_Area">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/32x32/gui/images/32x32/polygon.png</normaloff>:/icons/32x32/gui/images/32x32/polygon.png</iconset>
   </property>
   <property name="text">
    <string>Place Task Area</string>
   </property>
  </action>
  <action name="actionTest_Flight">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/48/gui/images/48/48px-Media-playback-start.svg.png</normaloff>:/icons/48/gui/images/48/48px-Media-playback-start.svg.png</iconset>
   </property>
   <property name="text">
    <string>Test Flight</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
